src/components/SettingsDialog/Participants/Participants.tsx

```diff
@@ -21,6 +22,11 @@ export const Participants = () => {
   const isModerator = useAppSelector((state) => state.participants!.self.role === "OWNER" || state.participants!.self.role === "MODERATOR");
   const self = useAppSelector((state) => state.participants!.self);
   const participants = useAppSelector((state) => [state.participants!.self, ...(state.participants?.others ?? [])]);
+  const existsAtLeastOneReadyUser = participants.some((p) => p.ready);
```

Comment:
thats a very long variable name

Reply:
yes that's true, but it's descriptive which is why I chose it :)